src/main/resources/reference.conf

```diff
@@ -25,10 +25,11 @@ clusterResources {
   proxyDockerCompose = "proxy-docker-compose.yaml"
   welderDockerCompose = "welder-docker-compose.yaml"
   proxySiteConf = "cluster-site.conf"
-  googleSignInJs = "google_sign_in.js"
-  jupyterGooglePlugin = "google_plugin_jupyter.js"
+  googleSignInJs = "google_sign_in.js" //TODO make naming convention for nbextension js files
```

Comment:
+1 on refactoring this config. Instead of individual configs for each extension, could have something like:
```
# the entrypoint
jupyterPlugin = "jupyter_plugin.js"
# the actual list of extensions which get templated into jupyter_plugin.js
jupyterExtensions = [ "list", "of", "jupyter", "extensions" ]
```
I haven't thought this through deeply and could probably wait for a separate refactoring ticket outside of this PR.

Reply:
+1 on doing this in another PR